bin/templates/project/__CLI__.xcodeproj/project.pbxproj

```diff
@@ -374,7 +374,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "cordova/lib/copy-www-build-step.sh";
+			shellScript = "node cordova/lib/copy-www-build-step.js";
```

Comment:
I thought about this earlier when the PR came in and I think its a good assumption since it's a Cordova project and the user is using the CLI. The scenario where a user would take the project from platforms/ios and run it stand-alone is miniscule. 

Not sure if this is supported fully but Xcode.app does include node.js under `/Applications/Xcode.app/Contents/Developer/usr/share/xcs/Node/bin/node`

You would dynamically get this path by using `xcode-select --print-path` then appending `/usr/share/xcs/Node/bin/node`